src/main/res/values/strings.xml

```diff
@@ -66,7 +66,6 @@
     <string name="no_satellites">No satellites</string>
     <string name="no_target">No target!</string>
     <string name="no_wherigo_cartridge_available">No Wherigo cartridge (*.gwc) is available.\nDownload or copy them into directory \"%1$s\". You can also set a path to custom directory in \"Settings - Main - Wherigo folder\". Make sure you granted the application permission to write to external storage and you have write permission in selected directory. Some newer devices don\'t allow writing to SD card.If you\'re not sure, set \"Wherigo folder\" automatically.</string>
-    <string name="not_enough_disk_space_x">Not enough free disk space on SDCard!\n(%1$s, free:%2$s)</string>
```

Comment:
It would be enough to delete the string in the english source file.
The change for the other languages will be then done by the crowdin integration.
You can let it as is.

Reply:
Thanks for the hint.